pubsub/kafka/metadata.yaml

```diff
@@ -0,0 +1,128 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: kafka
+version: v1
+status: stable
+title: "Apache Kafka"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-apache-kafka/
+capabilities:
+    - publish
+    - subscribe
+    - query
```

Comment:
I don't know the capanolities are correct or not, I could not find a reference to fill with this

Reply:
PubSUb components don't have capabilities, so please just remove the entire section